en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-config.md

```diff
@@ -0,0 +1,1774 @@
+        <th>Required</th>
+    </tr>
+    <tr>
+        <td>
+            Connection Name
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            A unique name to identify the connection.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            Yes
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Connection Type
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            The protocol used to connect to the MongoDB database.</br> 
+            <b>Possible values</b>: 
+            <ul>
+                <li>
+                    <b>STANDARD</b>:  The standard format of the MongoDB connection URI.
+                </li>
+                <li>
+                    <b>DSL</b>: The DNS-constructed seed list format of the MongoDB connection URI.
+                </li>
+                <li>
+                    <b>URI</b>: The complete connection URI containing the server details, credentials and the connection options.
+                </li>
+            </ul>
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            Yes
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Database
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            The name of the database in the MongoDB server.
+        <td>
+            -
+        </td>
+        <td>
+            Yes
+        </td>
+    </tr>
+</table>
+
+### URI connection configs
+
+<table>
+    <tr>
+        <th>Parameter Name</th>
+        <th>Type</th>
+        <th>Description</th>
+        <th>Default Value</th>
+        <th>Required</th>
+    </tr>
+    <tr>
+        <td>
+            Connection URI
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            The complete connection URI containing the server details, credentials and the connection options.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            Yes
+        </td>
+    </tr>
+</table>
+
+**Sample Configuration of URI configs**
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<localEntry key="uriConnection" xmlns="http://ws.apache.org/ns/synapse">
+    <mongodb.init>
+        <name>uriConnection</name>
+        <connectionType>URI</connectionType>
+        <connectionURI>mongodb+srv://server.example.com/?connectTimeoutMS=300000&authSource=aDifferentAuthDB</connectionURI>
+        <database>users</database>
+    </mongodb.init>
+</localEntry>
+```
+
+### Common configs for STANDARD and DSL types
+
+<table>
+    <tr>
+        <th>Parameter Name</th>
+        <th>Type</th>
+        <th>Description</th>
+        <th>Default Value</th>
+        <th>Required</th>
+    </tr>
+    <tr>
+        <td>
+            Replica Set
+        </td>
+        <td>
+            String
+        </td>
+        <td>
+            Specifies the name of the replica set, if the mongod is a member of a replica set.
```

Comment:
"If the mongoDB database is a member of a replica set, this parameter specifies the name of the replica set."

Not sure if this should be 'mongoDB' or 'mongod'

Reply:
```suggestion
If the mongod is a member of a replica set, this parameter specifies the name of the replica set.
```